tests/conftest.py

```diff
@@ -6,13 +9,45 @@
 pytest_plugins = "sphinx.testing.fixtures"
 
 
+def _set_localconf(app):
+    """
+    Load additional configuration for local environment
+
+    example: roots/test-root/localconf.json
+    {
+      "drawio_binary_path": "/Applications/draw.io.app/Contents/MacOS/draw.io",
+      "drawio_headless": false
+    }
+    """
+
+    local_conf_path = os.path.join(
+        os.path.dirname(__file__),
+        'roots',
+        'test-root',
```

Comment:
There can be different folders here (aside from just `test-root`, there could be a `test-format` folder).

I would prefer that this work between multiple folders; and actually the formats test be created in a  `test-format` folder (this makes it easier to know which test failed)